openedx/core/djangoapps/xmodule_django/models.py

```diff
@@ -181,6 +183,19 @@ class UsageKeyField(OpaqueKeyField):
     KEY_CLASS = UsageKey
 
 
+class UsageKeyWithRunField(UsageKeyField):
```

Comment:
@jibsheet: I think that we're the only ones who were early/crazy enough to run XML Modulestore courses, and I don't think those would work in today's LMS anyway. That code isn't completely gone, but it's being gradually picked away and hopefully refactored so that only the import/export relevant pieces remain.